packages/patternfly-4/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -330,7 +346,7 @@ class Wizard extends React.Component<WizardProps> {
         <FocusTrap focusTrapOptions={{ clickOutsideDeactivates: true }}>
           <Backdrop>
             <Bullseye>
-              <div {...rest} className={css(styles.wizard, className)} role="dialog" aria-modal="true" aria-labelledby={this.titleId} aria-describedby={description ? this.descriptionId : undefined}>
+              <div {...rest} style={{ width, height }} className={css(styles.wizard, isFullWidth && styles.modifiers.fullWidth, isFullHeight && styles.modifiers.fullHeight,className)} role="dialog" aria-modal="true" aria-labelledby={this.titleId} aria-describedby={description ? this.descriptionId : undefined}>
```

Comment:
Can you add Logic here to always apply isFullWith when there is a width set and alway set the isFullHeight when the height is set.

Reply:
should I remove 'isFullWidth/Height` from props?

Reply:
No, they should remain in the props.  When width or height are applied.the take precedence over the isFullWidth/Height.  we need to apply isFullWidth/Height for the scenario when a size that is bigger the max size is applied, the isFullWidth/Height modifiers will force the max to be overridden.